components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ImportUtils.java

```diff
@@ -640,6 +649,35 @@ public static String retrieveValidatedGraphqlSchemaFromArchive(String pathToArch
         }
     }
 
+    /**
+     * Retrieve graphql complexity information from the file and validate it with the schema
+     *
+     * @param pathToArchive Path to API archive
+     * @param schema        GraphQL schema
+     * @throws APIImportExportException If an error occurs while reading the file
```

Comment:
Shall we remove APIImportExportException since it has not been thrown?

Reply:
Fixed the code to throw APIManagementException